src/store/reducers/appReducer.js

```diff
@@ -1,18 +1,47 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import 'rxjs';
-import { createAction, createReducerScenario, createEpicScenario } from 'store/utilities';
+import { Observable } from 'rxjs';
+import { ConfigService } from 'services';
+import { schema, normalize } from 'normalizr';
+import { createSelector } from 'reselect';
+import update from 'immutability-helper';
+import {
+  createAction,
+  createReducerScenario,
+  createEpicScenario,
+  errorPendingInitialState,
+  pendingReducer,
+  errorReducer,
+  setPending,
+  toActionCreator,
+  getPending,
+  getError
+} from 'store/utilities';
 
 // ========================= Epics - START
+const handleError = fromAction => error =>
+  Observable.of(redux.actions.registerError(fromAction.type, { error, fromAction }));
+
 export const epics = createEpicScenario({
   /** Initializes the redux state */
   initializeApp: {
     type: 'APP_INITIALIZE',
     epic: () => [
-      // redux.actions.updateDeviceModels(),
+      epics.actions.fetchDeviceGroups(),
+      redux.actions.updateActiveDeviceGroup()
     ]
   },
 
+  /** Get the account's device groups */
+  fetchDeviceGroups: {
```

Comment:
It feels like this should be in deviceReducer.  The whole reason we called it "device" instead of "iotHubManager" was because we knew there would be mutliple services called to deal with devices.

Reply:
I put it here because it is a part of the app state. If you look at the current version of RM, the device groups are accessible/changeable from every view. Rules and actions, for example, care about the device groups and currently selected devices groups without references to the devices themselves.